docs/vz235_mellow/top_cover/top_cover.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: default
+title: 8.1 Top cover
+parent: 8. Top cover and doors panels
+grand_parent: Vz235 - Mellow Kit
+has_toc: false
+nav_order: 1
+has_children: false
+permalink: /vz235_mellow/top_cover/latches
+---
+
+# 8.1 Top Cover Assembly
+
+## Overview
+
+![Overview](../../assets/images/manual/vz235_printed/top_cover/top_cover_overview.png)
+
+## BOM
+
+| Material                    | Quantity | Notes                                                |
+| --------------------------- | -------- | ---------------------------------------------------- |
+| M3 x 8 BHCS                 | 8        |                                                      |
+| M3 Nut                      | 8        |                                                      |
+| M4 x 10 SHCS                | 8        |                                                      |
+| M4 hammer t-nut             | 8        |                                                      |
+| 2 x 8mm Magnet              | 8        |                                                      |
+| 395\*405\*3mm acrylic panel | 1        | Refer to the .step file for placement of the cutouts |
+
+## STL's
+
+| File name                      | Amount to print |
+| ------------------------------ | --------------- |
+| [Top cover holder]             | 4               |
+| [Top cover middle part]        | 4               |
+| [Top cover top part]           | 4               |
+
+### Step 1
+
+![Top cover holder assembly](../../assets/images/manual/vz235_mellow/top_cover/top_cover_holder_assembly.png)
+
+Assemble one (1) 2 x 8mm Magnets, two (2) of the M4 x 10 SHCS's and two (2) of the M4 hammer t-nuts to each of the Top cover holders.
+Ensure that the polarity of the magnets is the same in each holder.
+
+### Step 2
+
+![Top cover holder installation](../../assets/images/manual/vz235_mellow/top_cover/top_cover_holder_installation.png)
+
+Install one (1) assembled Top cover holder into each corner at the top of the machine with the magnet facing upwards.
+
+### Step 3
+
+![Top cover middle and top part assembly](../../assets/images/manual/vz235_mellow/top_cover/top_cover_holder_top_assembly.png)
+
+Assemble one (1) 2 x 8mm Magnet to the Top cover middle part. Ensure that the Magnet installed so it would be attracted to the magnet in the Top cover holder.
+Assemble one (1) Top cover middle cover middle part, one (1) Top cover top part, two (2) M3 x 8 BHCS and two (2) M3 nuts to each corner of the acrylic top panel.
+
+### Step 4
+
+![Handle Scews](../../assets/images/manual/vz235_mellow/top_cover/handle_screws.png)
```

Comment:
Could you change this picture's Alt text so it is not the same as step 6's picture? The alt text serves accessibility purposes, so a descriptive name is useful.

I personally use assistive technologies, so I've got an eye for things like these. :)

Reply:
Updated